README.md

```diff
@@ -331,6 +331,61 @@ and also by providing more type information to the Julia compiler.
   instead use `w.pymember(:member)` (for the `PyAny` conversion) or
   `w.pymember("member")` (for the raw `PyObject`).
 
+### Defining Python Classes
+
+`@pydef` creates a Python class. For instance,
+    
+    @pyimport numpy.polynomial as P
+
+    @pydef type Doubler <: P.Polynomial
+       # __init__ must return `nothing`
+       __init__(self, x=10) = (self[:x] = x; nothing) 
+       py_method1(self, arg1=5) = arg1 + 20  # the right-hand-side is Julia code
+       x2.get(self) = self[:x] * 2
+       x2.set!(self, new_x::Int) = (self[:x] = new_x / 2)
+    end
+
+    Doubler()[:x2]
+
+is equivalent to
+    
+    class Doubler(numpy.polynomial.Polynomial):
+       def __init__(self, x=10):
+          self.x = x
+
+       def pymethod1(self, arg1):
```

Comment:
`py_method` for consistency with the definition above?  Maybe call it `my_method1` instead, since `py_method` might give the wrong impression that there is something "pythonic" about this function which is actually in pure Julia.